spec/models/live_metric_spec.rb

```diff
@@ -32,7 +32,7 @@
   end
 
   it "#process_conditions" do
-    processed = LiveMetric.process_conditions(conditions)
+    processed = LiveMetric.process_conditions([conditions])
```

Comment:
why not create this as an array in the first place? (rather than create as a string and then convert to an array?)